Source/WebKit/UIProcess/API/Cocoa/WKWebViewTesting.mm

```diff
@@ -213,6 +215,13 @@ - (NSString *)_scrollingTreeAsText
     return coordinator->scrollingTreeAsText();
 }
 
+- (pid_t)_networkProcessIdentifier
+{
+    if (auto* process = _page->websiteDataStore().networkProcessIfExists())
+        return process->processIdentifier();
+    return 0;
```

Comment:
Isn't PID 0 reserved for kernel processes? Does/can that matter here?

Reply:
Oh, I was roughly following what `-_gpuProcessIdentifier` and `-_webProcessIdentifier` do already, in cases where their respective child processes are invalid or haven't been launched yet. It's possible we want to make them return something other than `0` (like returning `std::nullopt` or `nil` instead), but since it's all just used for testing, I don't think it's _too_ critical to change.

Reply:
Agreed, returning 0 seems benign in this situation.